cuqi/testproblem/_testproblem.py

```diff
@@ -148,69 +152,106 @@ class Deconvolution1D(BayesianProblem):
     :math:`\mathbf{x}` is a sharp (clean) signal and
     :math:`\mathbf{A}` is a convolution operator.
 
+    The convolution operator is defined by specifying a point spread function and
+    boundary conditions and is computed (matrix-free) via scipy.ndimage.convolve1D,
+    or using a matrix representation if the use-matrix flag is set to True.
+
+    The inputs are padded to fit the boundary conditions.
+    https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.convolve1d.html
+
     Parameters
     ------------
     dim : int
         size of the (dim,dim) deconvolution problem
 
-    kernel : string or ndarray
-        Determines type of the underlying kernel
-        'Gauss' - a Gaussian function
-        'sinc' or 'prolate' - a sinc function
-        'vonMises' - a periodic version of the Gauss function
-        ndarray - a custom kernel.
+    PSF : string or ndarray
+        | Determines type of the underlying point spread function (PSF).
+        | 'Gauss' - a Gaussian function
+        | 'sinc' or 'prolate' - a sinc function
+        | 'vonMises' - a periodic version of the Gauss function
+        | ndarray - a custom PSF.
```

Comment:
This is not for a full matrix right, but a PSF. Maybe state what size is expected here for a meaningful PSF?

Reply:
Good suggestion